### PR TITLE
update: 1279-blm-design-tweaks

### LINKED ISSUE #1279: [Carbon homepage]: BLM Banner design tweaks
## [Carbon homepage]: BLM Banner design tweaks

- [x] Change banner height from 48px to 32px 

- [x] Left align type to Carbon Design System 

- [x] Font size 14px; BLM is semi-bold ($heading-01); link is not semi-bold ($body-short-01)

- [x] Add @mjabbink as reviewer on this... haven't gotten his approval yet

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #1279 

Updated BLM banner to match design specs. 

#### Changelog

**Changed**

- [x] Change banner height from 48px to 32px 

- [x] Left align type to Carbon Design System 

- [x] Font size 14px; BLM is semi-bold ($heading-01); link is not semi-bold ($body-short-01)

- [x] Add @mjabbink as reviewer on this... haven't gotten his approval yet

### CODE REVIEW

**Comment by vpicone** [comment]
@jeanservaas [On their about page](https://eji.org/about/) they refer to themselves as the Equal Justice Initiative

Screen Shot 2020-06-10 at 6 10 38 PM

File: src/styles/Layout.module.scss [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
   top: 0;
   width: 100%;
   z-index: z("floating");
   display: flex;
   align-items: center;
-  justify-content: center;
-  height: 48px;
+  justify-content: flex-start;
+  height: 32px;
+  padding-left: 0.5rem;
```
Comment (vpicone) [inline src/styles/Layout.module.scss:32]:
```suggestion
  padding-left: $spacing-05;
```

**Comment by mjabbink** [comment]
@vpicone On their homepage they also refer to themselves as...

Screen Shot 2020-06-15 at 5 12 28 PM

—————

Screen Shot 2020-06-15 at 5 12 33 PM

—————

Screen Shot 2020-06-15 at 5 13 06 PM

—————

Screen Shot 2020-06-15 at 5 13 22 PM

**Comment by vpicone** [comment]
Their website url and logo convinced me the initialism would be the right way to go. 

Having “the” makes it a complete sentence as well.

**Pull request merged**